backend/googlecloudstorage/googlecloudstorage_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"testing"
 
 	"github.com/rclone/rclone/backend/googlecloudstorage"
+	"github.com/rclone/rclone/fstest"
```

Comment:
[linter wants you](https://github.com/rclone/rclone/runs/3263796986?check_suite_focus=true#step:14:23) to:
```
gofmt -w  ./backend/googlecloudstorage/googlecloudstorage_test.go
git commit -a -m 'gratify golang linter'
git push
```